platform/darwin/src/MGLFillExtrusionStyleLayer.h

```diff
@@ -0,0 +1,281 @@
+// This file is generated.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+#import "MGLVectorStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Controls the translation reference point.
```

Comment:
@1ec5 it might be — this language is copied from `*-translate-anchor` in every other layer type. I'll ticket this in that repo as a follow-up.